gitpod-io/gitpod
# Align experience of latest pinned and unpinned version

## Description


Make sure the first item of versions in ide config is added by `ide-service` and align with latest stable version's values

## Related Issue(s)

Fixes ENT-298, ENT-299

## How to test

- Latest stable version (check WORKSPACE.yaml) is pinnable on dashboard
- Pin version of IntelliJ with `2024.1.3`, it should be able to open workspace with
- Open this PR with Gitpod
- ~~Check `ide-config` if the latest stable versions' image info is the same as that first one in versions field~~
- Check logs of `ide-service` (see https://github.com/gitpod-io/gitpod/pull/19911/commits/147a140c355acfb559934ff0553323826902da4c) the latest stable versions' image info is the same as that first one in versions field

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - hw-ide-version</li>
	<li><b>🔗 URL</b> - <a href="https://hw-ide-version.preview.gitpod-dev.com/workspaces" target="_blank">hw-ide-version.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - hw-ide-version-gha.26453</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-hw-ide-version%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
